app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -34,6 +34,7 @@ def report_errors(error)
           Faraday::ClientError => :bad_request,
           Faraday::ConnectionFailed => :bad_request,
           Faraday::TimeoutError => :bad_request,
+          ActionController::InvalidAuthenticityToken => :bad_request,
         }[error.class] || :internal_server_error
```

Comment:
I think this is a good opportunity for a case/when instead of a hash, because case/when will allow subclasses. PR incoming!

Reply:
PR: https://github.com/18F/identity-idp/pull/8061